testing/test_transfer_learning.py

```diff
@@ -1,23 +1,24 @@
-
 import torch
```

Comment:
The test `test_inference` fails (typo: ivademed instead of ivadomed and a few `utils.` that has not been change to imed_utils) It is not related to changes made in this PR, but we probably should modify it :)